AsyncDisplayKit/Details/ASRangeController.h

```diff
@@ -160,13 +160,6 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)rangeController:(ASRangeController * )rangeController didEndUpdatesAnimated:(BOOL)animated completion:(void (^)(BOOL))completion;
 
 /**
- * Completed updates to cell node addition and removal.
- *
- * @param rangeController Sender.
- */
-- (void)didCompleteUpdatesInRangeController:(ASRangeController *)rangeController;
```

Comment:
I doubt this was used publicly, but do you think we should deprecate instead of just removing as it was exposed?